src/ethsign/CHANGELOG

```diff
@@ -4,15 +4,6 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.14.0] - 2020-05-29
```

Comment:
Why are we removing this?

If the changes have been reverted, we should mention that instead.